i18n/en-US.yml

```diff
@@ -26,18 +26,213 @@ _name: English
 # Component-specific messages (e.g. button captions)
 # are defined for each component under the 'components' category.
 components:
+  AccountSetupFinishPane:
+    message: You are ready to start planning your trips.
+  ActiveTripRenderer:
+    bodyText: "Trip is due to arrive at the destination at {arrivalTime}."
+    delayedHeading: "Trip is in progress and is delayed {deviationHumanDuration}!"
+    earlyHeading: "Trip is in progress and is arriving {deviationHumanDuration} earlier than expected!"
+    noDataHeading: Trip is in progress (no realtime updates available).
+    onTimeHeading: Trip is in progress and is about on time.
+  AfterSignInScreen:
+    mainTitle: Redirecting...
+    # For message, use the special <a>...</a> markup to automatically insert the
+    # corresponding link in the text.
+    message: "If the page doesn't load after a few seconds, <a>click here</a>."
+  BaseRenderer: 
+    bodyDefault: Unknown Trip State
+    lastCheckedDefaultText: Last checked time unknown
+    LastCheckedText: "Last checked: {formattedDuration} ago"
```

Comment:
Should be `lastCheckedText` (lower case `l`)

Reply:
addressed in https://github.com/opentripplanner/otp-react-redux/pull/408/commits/6c54a1e02879df30d2805cd8bc3d09152b79d7d7